crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -431,7 +431,7 @@ where
                     Ok(None) => {
                         // we don't have the block yet and the distance exceeds the allowed
                         // threshold
-                        return Some(state.finalized_block_hash)
+                        return Some(state.finalized_block_hash);
```

Comment:
```suggestion
                        return Some(state.finalized_block_hash)
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145